[CROSSORG-24] Update disclaimer language for Cross-Org Visibility (#26441)
DataDog/documentation: content/en/account_management/org_settings/cross_org_visibility.md

---

{{< callout url="#" btn_hidden="true">}}

>   Cross-organization visibility is in preview.

Reviewer: ```suggestion
  Cross-organization visibility is in Preview.
```

A _source_ organization exposes data to a _destination_ organization through an _organization connection_. The source and destination Datadog organizations must be in the same [account][1]. A source organization can have multiple destinations, and a destination organization can have multiple sources.

> Note that once the connection is established, the destination organization can query the source organization’s data in the same ways it can query its own data. This means that the source organization’s data—including sensitive data—may be queried and displayed as permitted by the destination organization’s access-control and other settings. This may include, for example, the destination organization’s ability to create [public Dashboards](/dashboards/sharing/shared_dashboards/#public-shared-dashboards) using the source organization’s data even if the source organization’s own settings do not permit the creation of public Dashboards.

Reviewer: ```suggestion
Once the connection is established, the destination organization can query the source organization's data in the same ways it can query its own data. This means that the source organization's data—including sensitive data—may be queried and displayed as permitted by the destination organization's access-control and other settings. This may include, for example, the destination organization's ability to create [public Dashboards](/dashboards/sharing/shared_dashboards/#public-shared-dashboards) using the source organization's data even if the source organization's own settings do not permit the creation of public Dashboards.
```

Reviewer: Could you update the link to match the formatting of the other links? i.e., have the destination URL listed at the bottom of the file. (For context, this is required due to how our translation pipeline works.)

Author: Fixed

> After you set up an organization connection, the exposed data is still stored in the source organization and does not move to the destination. Instead, the destination organization queries the data from the source. Connections do not duplicate the data, and do not incur extra charges. The destination organization can query source data from any time range supported by the source data, including prior to the date of the creation of the connection. If you remove the connection, the destination can no longer access any data from the source and any queries or Dashboards created from the source organization’s data may no longer function.

Reviewer: ```suggestion
After you set up an organization connection, the exposed data is still stored in the source organization and does not move to the destination. Instead, the destination organization queries the data from the source. Connections do not duplicate the data, and do not incur extra charges. The destination organization can query source data from any time range supported by the source data, including prior to the date of the creation of the connection. If you remove the connection, the destination can no longer access any data from the source and any queries or Dashboards created from the source organization's data may no longer function.
```

Author: Fixed
